apps/src/applab/designMode.js

```diff
@@ -585,6 +586,30 @@ function getUnsafeHtmlReporter(sanitizationTarget) {
   };
 }
 
+designMode.parseScreenFromLevelHtml = function (screenEl, allowDragging, prefix) {
+  var children = $(screenEl);
```

Comment:
It's a little weird to leave this named `children` when it will always be a single element.  In fact, much of the iteration in this method becomes redundant as it's used now.  Worth a refactor?